skip.go

```diff
@@ -0,0 +1,32 @@
+package slogx
+
+import (
+	"context"
+	"log/slog"
+	"runtime"
+	"time"
+)
+
+// LogSkip writes the location of called function to a record,
```

Comment:
As far as I understand a copy of slog.Log documentation `// Log calls Logger.Log on the default logger.` will not match to a LogSkip function. The way to mention that it works exactly as slog.Log will be better, I think.

Reply:
It may be better. I also think it may be better to refer original doc instead of copy&paste, but for a different reason: to avoid needs to keep this copy in sync with original.
But why do you think it's not match? They should do exactly the same, except for `skip` param.

Reply:
We don't have `Logger.Log` here and we use not `default logger` or `default handler`, we use the one that is fallback to the function.

Reply:
Okay, I see. My bad. I mention `slog.Log` while it should be `(*slog.Logger).Log`. It description fits much better, probably only what should be changed is first sentence by mentioning handler: "Log emits a log record **using handler** with the current time and the given level and message." and adding note about `skip` param.